Documentation/ReleaseNotes.md

```diff
@@ -69,6 +69,11 @@ We have introduced two helper components, [`UI_KeyboardInputField`](xref:Microso
 
 For more information, see - [Mixed Reality Keyboard Helpers](../Assets/MRTK/SDK/Experimental/MixedRealityKeyboard/README_MixedRealityKeyboard.md).
 
+**Grid Object Collection Alignment Options**
+
+<img src="https://user-images.githubusercontent.com/39840334/79289136-5c228780-7e7d-11ea-82b4-07959e42c3ed.gif" width="300" />
```

Comment:
Actually linking to githubusercontent is kinda neat - do you know what the retention story is for images and content that's there?

It could be better than saving stuff in our repo (i.e. storing binary blobs kinda blows in git)

Reply:
The bit of research I did seems to suggest that the content hosted on githubusercontent doesn't have an expiration time, so it seems like a good solution

Reply:
Da best